sound/soc/sof/topology.c

```diff
@@ -877,32 +877,47 @@ static int sof_widget_load_buffer(struct snd_soc_component *scomp, int index,
 {
```

Comment:
the second patch does not compile, please make sure each patch can be compiled to avoid breaking git bisect
/data/pbossart/ktest/sof-dev/sound/soc/sof/topology.c: In function ‘snd_sof_free_topology’:
/data/pbossart/ktest/sof-dev/sound/soc/sof/topology.c:2130:48: error: ‘struct snd_sof_dev’ has no member named ‘route_list’; did you mean ‘pcm_list’?